### DISCUSSION TITLE
How to set a colspan for tfoot

### DISCUSSION THREAD

**Discussion by caetanojssilveira** [bootstrap-vue/bootstrap-vue#7200] [Q&A]
How can I set a colspan to the tfoot th of the table?

Example:
![image](https://github.com/bootstrap-vue/bootstrap-vue/assets/58527868/6e606220-5d38-4255-b382-9dd89ad37f3e)

<template #foot(id) **colspan=4**>
                                Totais:
                            </template>
                            <template #foot(valor_requisicoes)>
                                {{ formataReal(valor_requisicoes) }}
                            </template>
                            <template #foot(valor_cancelado)>
                                {{ formataReal(valor_cancelado) }}
                            </template>
                            <template #foot(valor_lancamentos_manuais)>
                                {{ formataReal(valor_lancamentos_manuais) }}
                            </template>
                            <template #foot(valor_taxa)>
                                {{ formataReal(valor_taxa) }}
                            </template>
                            <template #foot(valor_repasse)>
                                {{ formataReal(valor_repasse) }}
                            </template>
                            <template #foot(data_tranf)>
                                <span></span>
                            </template>
                            <template #foot(nf)>
                                <span></span>
                            </template>
                            <template #foot(comp)>
                                <span></span>
                            </template>
                            <template #foot(status)>
                                <span></span>
                            </template>
                            <template #foot(acoes)>
                                <span></span>
                            </template>

**Comment by Hiws** [comment]
Use the `custom-foot` slot,  and create the `<tr>` yourself.

**Reply by caetanojssilveira** [reply]
Thank you very much, it worked!
![image](https://github.com/bootstrap-vue/bootstrap-vue/assets/58527868/46290bfa-288a-4615-b713-93459d2e07f5)
